src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -191,15 +191,17 @@ export default class DetailsBody extends React.Component<Props> {
                 <React.Fragment>
                   <SideHeader>
                     <span>{t('Alert Rule')}</span>
-
-                    <SideHeaderLink
-                      to={{
-                        pathname: `/settings/${params.orgId}/projects/${incident?.projects[0]}/alerts/metric-rules/${incident?.alertRule.id}/`,
-                      }}
-                    >
-                      <IconEdit />
-                      {t('View Rule')}
-                    </SideHeaderLink>
+                    {incident?.alertRule &&
+                      incident?.alertRule?.status !== AlertRuleStatus.SNAPSHOT && (
```

Comment:
This is a bit redundant since `?.` makes sure it exists, see: https://github.com/getsentry/frontend-handbook#optional-chaining

```suggestion
                    {incident?.alertRule?.status !== AlertRuleStatus.SNAPSHOT && (
```